src/main/perf/LineFileDocs.java

```diff
@@ -471,6 +486,29 @@ static Document cloneDoc(Document doc1) {
     return doc2;
   }
 
+  private String getRandomWord(String body) {
+    Random random = ThreadLocalRandom.current();
+    String[] bodyArr = body.split(" ");
+    if (bodyArr.length == 0) {
+      return "EMPTY_WORD";
```

Comment:
Same question here as below -- how often does this happen?  It's good I don't see it in the top N you listed in the PR op :)

Reply:
It's actually in the top 20:
`[RandomLabel.taxonomy, EMPTY_WORD] 49364`
Looks like there are around 50k occurrences of this happening. I can look more into why this is occurring though.